NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache Score
+Copyright [2020-2024] The Apache Software Foundation
```

Comment:
With Score being launched in 2022, should this be [2022-2024]?

Reply:
good point, let me fix that.

Reply:
done